delivery-receipts-api/src/test/java/uk/gov/di/authentication/deliveryreceiptsapi/lambda/NotifyCallbackHandlerTest.java

```diff
@@ -157,27 +158,34 @@ void shouldUpdateBulkEmailDeliveryReceiptsStatusForTermsAndConditionsEmailType()
         var templateID = IdGenerator.generate();
         when(configurationService.getNotificationTypeFromTemplateId(templateID))
                 .thenReturn(Optional.of(TERMS_AND_CONDITIONS_BULK_EMAIL));
+        when(configurationService.isBulkUserEmailEnabled()).thenReturn(true);
+        NotifyCallbackHandler handlerBulkEmailOn =
+                new NotifyCallbackHandler(
+                        cloudwatchMetricsService,
+                        configurationService,
+                        dynamoService,
+                        bulkEmailUsersService);
         var deliveryReceipt = createDeliveryReceipt(email, "delivered", "email", templateID);
         var event = new APIGatewayProxyRequestEvent();
         event.setHeaders(Map.of("Authorization", "Bearer " + BEARER_TOKEN));
         event.setBody(objectMapper.writeValueAsString(deliveryReceipt));
         String subjectId = "subject-id-1";
         UserProfile userProfile = new UserProfile().withEmail(email).withSubjectID(subjectId);
         when(dynamoService.getUserProfileByEmailMaybe(email)).thenReturn(Optional.of(userProfile));
-        handler.handleRequest(event, context);
+        handlerBulkEmailOn.handleRequest(event, context);
 
         verify(dynamoService).getUserProfileByEmailMaybe(email);
 
         verify(bulkEmailUsersService).updateDeliveryReceiptStatus(subjectId, "delivered");
     }
 
     @Test
-    void shouldNotUpdateBulkEmailDeliveryReceiptsStatusForTermsAndConditionsEmailType()
```

Comment:
I think we should keep a test like this - I think the name of this test is actually wrong (it should be "should not update Bulk Email Delivery Receipts Status for **Email Updated** type") but I think this is a valid test to keep in for when the switch is *on*, and we should just add your switch off test in on top of this